xml/jtag_registers.xml

```diff
@@ -117,14 +117,24 @@
             This operation should never result in a busy or error response.
             The address and data reported in the following Capture-DR
             are undefined.
+
+            When this operation leaves the values in \FdtmDmiAddress and
+            \FdtmDmiData unchanged.
```

Comment:
I think that there's a missing word (or extra word) here.  Also, it could be ambiguous whether this is unchanged from the value written by the nop command or unchanged from a previous value (where the command is parsed to determine if it's a nop and only if it's not a nop will the Update-DR update those other fields).  Since those are R/W fields and they would not be updated by hardware on a nop, my assumption has been that the required behavior is the former: the value seen on a read is the value written by the nop command.

Reply:
Yes, no grammar.

In spike, scanning NOP results in both data and address being updated.
In a SiFive FPGA image, scanning NOP results in data and address being zero'd out.

Since it doesn't matter for anything that I can think of, I think we should just say it's undefined.

Reply:
Unspecified, I mean.

Reply:
Having DmiData reflect a possible value would not be a backwards-compatible change, and we're trying not to functionally change the spec anymore. Really what you're asking for is a separate dmi.op that performs a read followed by a write. That's something we could discuss, but not in this PR.